staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -179,8 +179,32 @@ type ValidatingAdmissionPolicySpec struct {
 	// +listType=atomic
 	// +optional
 	AuditAnnotations []AuditAnnotation `json:"auditAnnotations,omitempty" protobuf:"bytes,5,rep,name=auditAnnotations"`
+
+	// MatchConditions is a list of conditions that must be met for a request to be validated.
+	// Match conditions filter requests that have already been matched by the rules,
+	// namespaceSelector, and objectSelector. An empty list of matchConditions matches all requests.
+	// There are a maximum of 64 match conditions allowed.
+	//
+	// If a parameter object is provided, it can be accessed via the `params` handle in the same
+	// manner as validation expressions.
```

Comment:
When this graduates (to beta? stable?) we should probably add this to the expression docs comment directly (noting that it doesn't apply to webhooks). Not sure if we want to do that while CEL admission is still alpha though.

Reply:
ack, leaving this as-is for now